vagrant-kubernetes-istio/install_prereqs_debian.sh

```diff
@@ -32,7 +32,8 @@ fi
 # Install virtualbox.
 echo "Checking and Installing Virtualbox as required"
 virtualbox --help > /dev/null
```

Comment:
should this be a version check instead?

so you don't have to always run "sudo apt-get install virtualbox" below?